### ISSUE TITLE
Depending on how atom is launched pandoc might not be found

### ISSUE THREAD

**Issue by geekpete** [lsegal/atom-rst-preview#27]
Seems to be this atom bug where the user's true path config is not loaded correctly depending on how atom is launched:
https://github.com/AtomLinter/Linter/issues/150

This can manifest with this plugin as an error like:
```
'pandoc' could not be spawned. 
Is it installed and on your path? 
If so please open an issue on the package spawning the process.
```

This error is avoided if launching "atom" from the terminal.
But if you then pin the atom icon to the OSX dock, close atom then relaunch it from the OSX dock, the pandoc not found error will occur due to issues loading /usr/local/bin into the path.

Until this atom bug is fixed, a workaround that worked for me was to add the following config to the ~/.atom/init.coffee file the relaunch from the OSX dock:
```
process.env.PATH = ["/usr/bin",
                    "/usr/local/bin",
                    ].join(":")
```

This plugin's README file could mention this issue (for OSX users anyway) and suggest the workaround so that users can avoid hitting this path issue with the plugin.

Testing was just to copy/paste this content into a new tab and hitting ctrl-shift-r:
https://raw.githubusercontent.com/ralsina/rst-cheatsheet/master/rst-cheatsheet.rst

Cheers.

**Comment by geekpete** [comment]
Added a pull request to for the README file:
https://github.com/lsegal/atom-rst-preview/pull/28

**Comment by lsegal** [comment]
Fixed by #28
